### PR TITLE
feat: add language metrics script

### PR DESCRIPTION
**Pull request by mitchell-phillips**

### CODE REVIEW

File: org-disco/language-metrics.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import click
```
Comment (mrparkers) [inline org-disco/language-metrics.py:1]: should this be in the `requirements.txt`?

**Review by mrparkers — COMMENTED** [review]
A couple of other comments:

- I think the output should try and be more clear about documenting which repositories were found and which languages they have.  I don't think displaying only a simple count is enough to act on.  Maybe we can store the repository names within the map and print those along with the language numbers?
- I really think we should be using the [languages API endpoint](https://docs.github.com/en/rest/repos/repos#list-repository-languages) to specifically ask GitHub which languages are in a repository.  If we don't do this, we're going to return incorrect metrics about repositories that have code using more than one language.
- We can probably filter out the "none" results too

**Comment by mitchell-phillips** [comment]
> A couple of other comments:
> 
> * I think the output should try and be more clear about documenting which repositories were found and which languages they have.  I don't think displaying only a simple count is enough to act on.  Maybe we can store the repository names within the map and print those along with the language numbers?
> * I really think we should be using the [languages API endpoint](https://docs.github.com/en/rest/repos/repos#list-repository-languages) to specifically ask GitHub which languages are in a repository.  If we don't do this, we're going to return incorrect metrics about repositories that have code using more than one language.
> * We can probably filter out the "none" results too

@mrparkers Using the language api, should we be focusing and adding metrics around all language usage or just the primary?  For example: https://api.github.com/repos/liatrio/elasticsearch/languages

**Comment by mrparkers** [comment]
Hmm... There's probably a good way for us to tell whether or not it's a "primary" language, or just a one-off script.  Maybe we could log the language if at least 10% of the code uses that language, or something similar to that.

**Pull request merged**